### DISCUSSION TITLE
Nvim tree hides statusbar

### DISCUSSION THREAD

**Discussion by Luxvao** [nvim-tree/nvim-tree.lua#2429] [Q&A]
Is it possible to keep the status bar when the focus is on nvim tree?
![image](https://github.com/nvim-tree/nvim-tree.lua/assets/95533447/67ecc3a4-bad0-4a1b-badf-a5ac138481db)
When I switch to the other buffer I get this:
![image](https://github.com/nvim-tree/nvim-tree.lua/assets/95533447/c615a7d0-c233-4fe4-a37b-7518a37d198e)

**Comment by alex-courtis** [comment]
Looks like you're using lualine. It has some basic nvim-tree integrations.

You can customise. See my configuration which shows filetype: https://github.com/alex-courtis/arch/blob/56948c23dce6ddf7c0c2dc188c29fc511ccdc49f/config/nvim/lua/amc/plugins/lualine.lua#L112

**Reply by Luxvao** [reply]
I should have mentioned. I use feline not lualine. Thanks anyway!

**Reply by alex-courtis** [reply]
Today I learned about Feline.

Please post when you come up with a configuration that looks nice.

**Reply by Luxvao** [reply]
Okay I will!
